### PR TITLE
add check to verify that yum is updated

### PR DESCRIPTION
**Pull request by LiquidPT**

### CODE REVIEW

File: tests/run-tests.sh [hunk 1]
```diff
@@ -443,6 +443,14 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is updated
+case ${distro} in
+    Ubuntu*) sudo apt-get -q --assume-no update;;
+    CentOS* | AlmaLinux*) sudo yum update -n;;
```
Comment (abhamidipati0614) [inline tests/run-tests.sh:449]: Pls use --assumeno
yum update --assumeno

Comment (abhamidipati0614) [inline tests/run-tests.sh:449]: pls use --setopt tsflags=test flag
`yum update -y --setopt tsflags=test`
and then clean the cache using `yum clean packages`

File: tests/run-tests.sh [hunk 2]
```diff
@@ -443,6 +443,14 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is updated
```
Comment (abhamidipati0614) [inline tests/run-tests.sh:446]: we are checking for updates on all the distros. pls change yum to package as "verify if package update works"

**Review by abhamidipati0614 — CHANGES_REQUESTED** [review]

File: tests/run-tests.sh [hunk 3]
```diff
@@ -443,6 +443,17 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is correctly updated
```
Comment (abhamidipati0614) [inline tests/run-tests.sh:446]: pls correct this... it not only pertains to yum but apt as well

File: tests/run-tests.sh [hunk 4]
```diff
@@ -443,6 +443,17 @@ check_exit_code() {
     fi
 }
 
+# verify that yum is correctly updated 
+case ${distro} in
+    Ubuntu*) sudo apt-get -q --assume-no update;;
+    CentOS* | AlmaLinux*) 
+        yum update -y --setopt tsflags=test
```
Comment (abhamidipati0614) [inline tests/run-tests.sh:450]: semicolons at the end of each line inside a case condition

**Review by abhamidipati0614 — CHANGES_REQUESTED** [review]

File: tests/run-tests.sh (hunk 4 above)
Comment (abhamidipati0614) [reply tests/run-tests.sh:450]: Oh! you also need to run it as sudo

Comment (LiquidPT) [reply tests/run-tests.sh:450]: > semicolons at the end of each line inside a case condition

I think you only need the double semicolons at the end of the condition. No other semicolons at the end of each line withing the condition. That's how I found it in the examples.

**Review by abhamidipati0614 — APPROVED** [review]

**Pull request merged**